Node.gitignore

```diff
@@ -3,6 +3,10 @@ logs
 *.log
 npm-debug.log*
 
+# .DS_Store
```

Comment:
copy that, however isn't it true that only one `.gitignore` is available during the creation of a repository, and in which case OS specific rules may be included?